test/test.js

```diff
@@ -0,0 +1,177 @@
+const cont = require('../js/actions');
```

Comment:
I would expect the controller to get data from somewhere. You are kind of using the data in the controller instead of some stub data

Reply:
If I'm using the same data for testing, wouldn't adding the same data to test.js as stub be a bit irrelevant?

Reply:
Currently you are using a local array.
In a real project, you will probably get the data from some service / a DB / async`ly.

So, you won't like your tests to change the DB or something. You would have functions that are as independent as possible and that are testible (even w/o a DB connection)